SYCL/Basic/buffer/buffer.cpp

```diff
@@ -394,7 +394,7 @@ int main() {
       myQueue.submit([&](handler &cgh) {
         accessor<int, 2, access::mode::write, access::target::device,
                  access::placeholder::false_t>
-            B(Buffer, cgh, range<2>(20, 20), id<2>(10, 10));
+            B(Buffer, cgh, range<2>(10, 10), id<2>(10, 10));
```

Comment:
These changes look unrelated to SYCL2020 errc conformance. Can we remove them from this PR?

Reply:
One of the conformance checks is that out-of-bounds range args should trigger an exception in accessor construction. I added that.  Without this change, this test triggers that error.